cogs/zalgo.py

```diff
@@ -22,6 +22,16 @@ def __init__(self, bot):
 
     @commands.command()
     async def zalgo(self, *, text: str):
+        list = ["Really? this is over 1000 letters long!", "Hmmm... Are you trying to launch a DOS attack?",
+                "I can't proccess this much data, I'm on an Intel 4004", "This is longer than my Thesis...",
+                "This is worse than if someone launched a Low Orbit Ion Cannon at me, keep your message shorter",
+                "```Error 404: keep your message shorter```", "I'm out of RAM, keep your message shorter so i can process this",
+                "You know, Discord has a limit of 2k characters, you just exceeded half of that, keep it shorter please"]
+        
+        while len(text) >= 1000:
+            await self.bot.say(random.choice(list)) # noticed choice wasn't being used so i'm gonna use it
+            break

```

Comment:
This break serves no purpose here: you'll still end up trying to send the message, where it'll error out.  Just `return` here and as I state above, change it to an `if` statement.